pkg/pool-weighted/contracts/smart/ManagedPool.sol

```diff
@@ -48,6 +48,8 @@ import "./WeightCompression.sol";
  * IL-protection, and more.
  */
 contract ManagedPool is BaseWeightedPool, ReentrancyGuard {
+    // LiquidityBootstrappingPool change their weights over time: these periods are expected to be long enough (e.g.
+    // days) that any timestamp manipulation would achieve very little.
```

Comment:
```suggestion
    // LiquidityBootstrappingPools change their weights over time: these periods are expected to be long enough
    // (e.g., days) that any timestamp manipulation would achieve very little.
```

Reply:
Huh, interesting. [According to this article](https://jakubmarian.com/comma-after-i-e-and-e-g/), this is a British vs American thing. We're doing brit style throughout the entire process, so I'd keep things consistent here and go with the Queen's grammar.